framework/mesh/mesh.h

```diff
@@ -79,3 +49,5 @@ std::shared_ptr<MeshContinuum> GetCurrentMesh();
 #include "framework/mesh/mesh_matrix3x3.h"
 #include "framework/mesh/mesh_face.h"
 #include "framework/mesh/mesh_edge_loops.h"
+#include "framework/mesh/cell/cell.h"
+#include "framework/mesh/logical_volume/logical_volume.h"
```

Comment:
@andrsd I didn't like adding this. maybe we can talk early next week

Reply:
Well,  `mesh.h` needs some audit. This interleaving if code and #include needs to be fixed. I think a lot of code adds `mesh.h` to it masks some dependency "issues".

You may want to open and issue for this audit...

Reply:
For this PR this is ok...

Reply:
Thanks. I have created #434 and will resolve this discussion.